partcad/src/partcad/runtime_python.py

```
@@ -39,14 +91,127 @@
+                            ]
+                        )
+                # Install of the dependencies into the venv environment
+                for dep in session["deps"]:
+                    self.ensure_onced_locked(dep, path=session["path"])
 
-    async def run(self, cmd, stdin="", cwd=None):
+        python_path = self.get_venv_python_path(session, path)
+        cmd = [python_path, *self.python_flags, *cmd]
         pc_logging.debug("Running: %s", cmd)
+        # pc_logging.debug("stdin: %s", stdin)
+        p = subprocess.Popen(
```

Ruff (0.8.2)

`subprocess` call: check for execution of untrusted input

(S603)